Compile time error with cmake debug flags

When compiling the current `dev` branch with the following CMAKE flags for debugging a compile time error occurs.   

```cmake
CUDA_KEEP_FILES ON
CUDA_SHOW_CODELINES ON
CUDA_SHOW_REGISTER ON
PMACC_BLOCKING_KERNEL ON
CUDA_NVCC_FLAGS -g;-G
```

```
[ 58%] Built target mpiInfo
/home/.../include/particles/bremsstrahlung/ScaledSpectrum.tpp(41): error: 
identifier "picongpu::particles::bremsstrahlung::electron::MIN_ENERGY" is undefined in device code

/home/.../include/particles/bremsstrahlung/ScaledSpectrum.tpp(42): error: 
identifier "picongpu::particles::bremsstrahlung::electron::MAX_ENERGY" is undefined in device code

2 errors detected in the compilation of "/home/.../nvcc_tmp/main.cpp2.i".
CMake Error at picongpu_generated_main.cu.o.cmake:266 (message):
  Error generating file
  /home/garten70/build/build_picongpu/CMakeFiles/picongpu.dir//./picongpu_generated_main.cu.o
```
This leads to the following file implemented by @Heikman 
```c++
/** constructor
 *
 * @param linInterpCursor
 */
HDINLINE LookupTableFunctor::LookupTableFunctor(LinInterpCursor linInterpCursor)
    : linInterpCursor(linInterpCursor)
{
    this->lnEMin = math::log(electron::MIN_ENERGY);
    this->lnEMax = math::log(electron::MAX_ENERGY);
}
```

--- a/src/picongpu/include/particles/bremsstrahlung/ScaledSpectrum.tpp
+++ b/src/picongpu/include/particles/bremsstrahlung/ScaledSpectrum.tpp
@@ -17,6 +17,7 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include "simulation_defines.hpp"
 #include "cuSTL/container/HostBuffer.hpp"
 
 namespace picongpu
@@ -37,8 +38,10 @@ namespace detail
 HDINLINE LookupTableFunctor::LookupTableFunctor(LinInterpCursor linInterpCursor)
     : linInterpCursor(linInterpCursor)
 {
-    this->lnEMin = math::log(electron::MIN_ENERGY);
-    this->lnEMax = math::log(electron::MAX_ENERGY);
+    float_X const lnEMinTmp( electron::MIN_ENERGY );
+    float_X const lnEMaxTmp( electron::MAX_ENERGY );
+    this->lnEMin = math::log( lnEMinTmp );
+    this->lnEMax = math::log( lnEMaxTmp);
 }
 
 /** scaled differential cross section

